tools/cockpit.spec

```diff
@@ -56,8 +56,8 @@
 %define build_subscriptions 1
 %endif
 
-# cockpit-kubernetes is RHEL 7 only, and 64 bit arches only
-%if 0%{?fedora} || (0%{?rhel} >= 7 && 0%{?rhel} < 8)
+# cockpit-kubernetes is RHEL 7 and Fedora < 30 only, and 64 bit arches only
+%if (0%{?fedora} && %{?fedora} < 30) || (0%{?rhel} >= 7 && 0%{?rhel} < 8)
```

Comment:
error: unmatched (

Reply:
I've stared at it for two minutes and can't see the missing (, but apparently something is wrong on rhel/centos 7. I'll have a look.

Reply:
Ah, missing `0` :) Fixed.